crates/oxc_linter/src/utils/jest/parse_jest_fn.rs

```diff
@@ -597,3 +608,1302 @@ const VALID_JEST_FN_CALL_CHAINS: [[&str; 4]; 51] = [
+    "suite.skipIf.sequential.todo",
+    "suite.skipIf.shuffle.skip",
+    "suite.skipIf.shuffle.only",
+    "suite.skipIf.shuffle.concurrent",
+    "suite.skipIf.shuffle.sequential",
+    "suite.skipIf.shuffle.todo",
+    "suite.skipIf.todo.skip",
+    "suite.skipIf.todo.only",
+    "suite.skipIf.todo.concurrent",
+    "suite.skipIf.todo.sequential",
+    "suite.skipIf.todo.shuffle",
+    "suite.runIf.skip.only",
+    "suite.runIf.skip.concurrent",
+    "suite.runIf.skip.sequential",
+    "suite.runIf.skip.shuffle",
+    "suite.runIf.skip.todo",
+    "suite.runIf.only.skip",
+    "suite.runIf.only.concurrent",
+    "suite.runIf.only.sequential",
+    "suite.runIf.only.shuffle",
+    "suite.runIf.only.todo",
+    "suite.runIf.concurrent.skip",
+    "suite.runIf.concurrent.only",
+    "suite.runIf.concurrent.sequential",
+    "suite.runIf.concurrent.shuffle",
+    "suite.runIf.concurrent.todo",
+    "suite.runIf.sequential.skip",
+    "suite.runIf.sequential.only",
+    "suite.runIf.sequential.concurrent",
+    "suite.runIf.sequential.shuffle",
+    "suite.runIf.sequential.todo",
+    "suite.runIf.shuffle.skip",
+    "suite.runIf.shuffle.only",
+    "suite.runIf.shuffle.concurrent",
+    "suite.runIf.shuffle.sequential",
+    "suite.runIf.shuffle.todo",
+    "suite.runIf.todo.skip",
+    "suite.runIf.todo.only",
+    "suite.runIf.todo.concurrent",
+    "suite.runIf.todo.sequential",
+    "suite.runIf.todo.shuffle",
+    "suite.skip.only.each",
+    "suite.skip.concurrent.each",
+    "suite.skip.sequential.each",
+    "suite.skip.shuffle.each",
+    "suite.skip.todo.each",
+    "suite.only.skip.each",
+    "suite.only.concurrent.each",
+    "suite.only.sequential.each",
+    "suite.only.shuffle.each",
+    "suite.only.todo.each",
+    "suite.concurrent.skip.each",
+    "suite.concurrent.only.each",
+    "suite.concurrent.sequential.each",
+    "suite.concurrent.shuffle.each",
+    "suite.concurrent.todo.each",
+    "suite.sequential.skip.each",
+    "suite.sequential.only.each",
+    "suite.sequential.concurrent.each",
+    "suite.sequential.shuffle.each",
+    "suite.sequential.todo.each",
+    "suite.shuffle.skip.each",
+    "suite.shuffle.only.each",
+    "suite.shuffle.concurrent.each",
+    "suite.shuffle.sequential.each",
+    "suite.shuffle.todo.each",
+    "suite.todo.skip.each",
+    "suite.todo.only.each",
+    "suite.todo.concurrent.each",
+    "suite.todo.sequential.each",
+    "suite.todo.shuffle.each",
+    "suite.skipIf.skip.each",
+    "suite.skipIf.only.each",
+    "suite.skipIf.concurrent.each",
+    "suite.skipIf.sequential.each",
+    "suite.skipIf.shuffle.each",
+    "suite.skipIf.todo.each",
+    "suite.runIf.skip.each",
+    "suite.runIf.only.each",
+    "suite.runIf.concurrent.each",
+    "suite.runIf.sequential.each",
+    "suite.runIf.shuffle.each",
+    "suite.runIf.todo.each",
+    "xtest",
+    "xtest.each",
+    "xit",
+    "xit.each",
+    "fit",
+    "xdescribe",
+    "xdescribe.each",
+    "fdescribe"
+};
```

Comment:
This is huge! Any way we can shrink this down?

Reply:
Sorry for the late, I'll check this.

Reply:
Updated.